.github/workflows/benchmarks.yml

```diff
@@ -51,11 +51,13 @@ jobs:
           ${{ runner.os }}-pip-
 
     - name: Install ubuntu dependencies
+      env:
+        DEBIAN_FRONTEND=noninteractive
```

Comment:
```suggestion
        DEBIAN_FRONTEND: noninteractive
```

Not sure why this is needed? But i think this should be like ^

Reply:
Oh, just saw the chat in IPS channel. Sounds like we could just drop it anyway?

Reply:
Thanks - is this how we set env variables?

Reply:
Yeah, the Benchmark workflow didn't even run before. Obviously because of wrong syntax.